apps/keyboard/js/views/layout_page_view.js

```diff
@@ -137,13 +137,6 @@ LayoutPageView.prototype.render = function render() {
     content.appendChild(kbRow);
   }).bind(this));
 
-  // If this layout does not require different rendering for lowercase state,
-  // we default to uppercase rendering -- this class will tell CSS file to
-  // never toggle button label <span> elements.
-  if (!layout.secondLayout) {
```

Comment:
Not sure if we should remove all `secondLayout` property in the layout file. Maybe do it in a follow-up patch when this patch merged and sticks?